src/app/music/VinylLore.jsx

```
@@ -0,0 +1,122 @@
+      },
+      onStart: () => {
+         const stateObj = {
+            state: active ? 'active' : 'idle',
+            album: album_object,
+         };
+         if (active) {
+            play();
+         } else {
+            end();
+         }
+         onAnimationStart && onAnimationStart(statifeObj);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)